setup.py

```diff
@@ -23,7 +23,10 @@
         'python-telegram-bot',
         'picamera',
         'RPi.GPIO',
+        'opencv-python',
+        'imutils',
         'numpy',
+        'scipy',
```

Comment:
Is this used?

Reply:
I checked and no, its not used. I will remove this